src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/FreeTextType.java

```diff
@@ -0,0 +1,31 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Arrays;
+
+@RequiredArgsConstructor
+@Getter
+public enum FreeTextType {
+    CLINICAL_INFO("CID"),
+    COMPLEX_REFERENCE_RANGE("CRR"),
+    INVESTIGATION_RESULT("RIT"),
+    REFERENCE_POPULATION_DEFINITION("RPD"),
+    SERVICE_PROVIDER_COMMENT("SPC");
+
+    private final String qualifier;
+
+    public String getKeyQualifier() {
+        return FreeTextSegment.KEY
```

Comment:
I'm not a big fan of this, I feel we're coupling this to segment parsing. Also, it's not in line with the other enum classes we've created.

Reply:
Okey dokey